### DISCUSSION TITLE
Adding images to disciples

### DISCUSSION THREAD

**Discussion by fellipesg** [DiscipleTools/disciple-tools-theme#2396] [Q&A]
Hey guys!
I was recently introduced to this theme and I`m wondering if there is a way for we to add photos(like profile pictures) to the disciples? Is there a way we can tweak the theme in order to store and show a profile picture for each disciple?

**Comment by corsacca** [comment]
Totally with you.
The big issue is that wordpress doesn't store images securely. So we're investigating leveraging amazon s3 (or other storage providers) for media. 
https://github.com/DiscipleTools/disciple-tools-theme/issues/2395

**Comment by corsacca** [comment]
This was implemented in:
https://disciple.tools/news/theme-release-v1-62/
and upgraded in
https://disciple.tools/news/disciple-tools-v1-73/
